app/src/main/java/com/example/hannahpaulson/popularmovies/ui/adapters/ListOfPosterAdapter.java

```diff
@@ -0,0 +1,58 @@
+package com.example.hannahpaulson.popularmovies.ui.adapters;
+
+import android.content.Context;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+
+import com.example.hannahpaulson.popularmovies.R;
+import com.example.hannahpaulson.popularmovies.data.datapojo.Movie;
+import com.squareup.picasso.Picasso;
+
+import java.util.List;
+
+public class ListOfPosterAdapter extends ArrayAdapter {
+
+    private Context context;
+    private List<Movie> movies;
+
+    public ListOfPosterAdapter(Context context, List<Movie> movies) {
+        super(context, 0, movies);
+        this.context = context;
+        this.movies = movies;
+    }
+
+    @Override
+    public int getCount() {
+        return movies.size();
+    }
+
+    @Override
+    public long getItemId(int position) {
+        return 0;
+    }
+
+    @Override
+    public Object getItem(int position) {
+        return null;
```

Comment:
Throw a runtimeexception error beacuse we prefer to use movies because it has a type